playbooks/linode.yml

```diff
@@ -49,5 +49,6 @@
   roles:
     - genesis-linode
 
+- include: linode-headers.yml
```

Comment:
Rather than including this here, you should just include it unconditionally on the wireguard playbook. Sure some providers might already provide it, but others might not, so best to just have it in there anyway. More robust that way.